### PR TITLE
F upgrade storybook migrate dp autocomplete

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://github.com/demos-europe/demosplan-ui/pull/379

- Migrate DpAutocomplete stories to v7.
- remove Translator.trans from DpAutocomplete

### CODE REVIEW

File: src/components/DpAutocomplete/DpAutocomplete.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+export default meta
+type Story = StoryObj<typeof DpAutocomplete>
+
+export const Default: Story = {
+    args: {
+        height: '28px',
+        label: 'label',
+        routeGenerator: function () {},
```
Comment (salisdemos) [inline src/components/DpAutocomplete/DpAutocomplete.stories.tsx:21]: this function should return a (empty) string
```suggestion
        routeGenerator: () => '',
```

File: src/components/DpAutocomplete/DpAutocomplete.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    }
+}
+
+export default meta
+type Story = StoryObj<typeof DpAutocomplete>
+
+export const Default: Story = {
+    args: {
+        height: '28px',
+        label: 'label',
```
Comment (salisdemos) [inline src/components/DpAutocomplete/DpAutocomplete.stories.tsx:19]: we don't need to pass the default values.

File: src/components/DpAutocomplete/DpAutocomplete.vue [hunk 3]
```diff
@@ -55,7 +56,7 @@ export default {
     placeholder: {
       type: String,
       required: false,
-      default: () => Translator.trans('search')
+      default: () => de.placeholder
```
Comment (salisdemos) [inline src/components/DpAutocomplete/DpAutocomplete.vue:59]: If You do things like this here, I would remove the unnecesarry method from label -> default too...
```
    label: {
      type: String,
      required: false,
      default: 'label'
    },
```

File: src/components/DpAutocomplete/DpAutocomplete.stories.mdx [hunk 4]
```diff
@@ -2,7 +2,7 @@ import { Meta } from '@storybook/addon-docs'
 import DpAutocomplete from './DpAutocomplete'
 
 <Meta
-  title="Components/Autocomplete"
+  title="Components/DpAutocomplete"
```
Comment (salisdemos) [inline src/components/DpAutocomplete/DpAutocomplete.stories.mdx:5]: A general Question: Do we want to have the `Dp`-Prefix in the Title/Navigation or not?

**Review by salisdemos — APPROVED** [review]

File: src/components/DpAutocomplete/DpAutocomplete.vue (hunk 3 above)
Comment (ahmad-demos) [reply src/components/DpAutocomplete/DpAutocomplete.vue]: Done in https://github.com/demos-europe/demosplan-ui/pull/396/commits/3ec68451d9f289f5e26008050ffc3ee52df18642

File: src/components/DpAutocomplete/DpAutocomplete.stories.tsx (hunk 2 above)
Comment (ahmad-demos) [reply src/components/DpAutocomplete/DpAutocomplete.stories.tsx]: Done in https://github.com/demos-europe/demosplan-ui/pull/396/commits/c6bc9c28795611cfd6215e05c3ef69b2406b3649

File: src/components/DpAutocomplete/DpAutocomplete.stories.mdx (hunk 4 above)
Comment (ahmad-demos) [reply src/components/DpAutocomplete/DpAutocomplete.stories.mdx:5]: Yes we need, see in https://demosdeutschland.slack.com/archives/C5BA8ABB2/p1690353485332819?thread_ts=1690293841.102939&cid=C5BA8ABB2

When we make without `DP`-Prefix name, then the Docs will be separate from the default!

Comment (salisdemos) [reply src/components/DpAutocomplete/DpAutocomplete.stories.mdx:5]: We only need the same Title in both files I guess. It shouldn't have anything todo with the Dp if I am right.

Comment (ahmad-demos) [reply src/components/DpAutocomplete/DpAutocomplete.stories.mdx]: Yes you are right
Done in https://github.com/demos-europe/demosplan-ui/pull/396/commits/c7bab32955945945592fc6d128526305432cd559
Thanks!

Comment (spiess-demos) [reply src/components/DpAutocomplete/DpAutocomplete.stories.mdx]: thanks for changing that!

File: src/components/shared/translations.js [hunk 5]
```diff
@@ -5,6 +5,7 @@ const de = {
   errorMandatoryFieldsIntro: 'Bitte korrigieren Sie Ihre Eingabe in folgenden Feldern: ',
   errorMandatoryFieldsOutro: '. Erst dann können Sie Ihre Angaben speichern.',
   errorMandatoryFieldsDefault: 'Bitte füllen Sie alle Pflichtfelder korrekt aus. Erst dann können Sie Ihre Angaben speichern.',
+  placeholder: "Suche"
```
Comment (spiess-demos) [inline src/components/shared/translations.js:8]: I'd suggest to specify both the key and the value. "Suche" is too vague, and so is "placeholder".

```suggestion
  placeholderAutoSuggest: "Suchbegriff..."
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: src/components/shared/translations.js (hunk 5 above)
Comment (ahmad-demos) [reply src/components/shared/translations.js]: Done also in https://github.com/demos-europe/demosplan-ui/pull/396/commits/c176fd60bb45475146988f0460f8d82c0a3cc27d

**Review by spiess-demos — APPROVED** [review]
lgtm!

**Pull request merged**